hbase-client/src/main/java/org/apache/hadoop/hbase/client/RpcRetryingCallerFactory.java

```diff
@@ -33,17 +33,24 @@ public class RpcRetryingCallerFactory {
   private final ConnectionConfiguration connectionConf;
   private final RetryingCallerInterceptor interceptor;
   private final int startLogErrorsCnt;
+  private final MetricsConnection metrics;
 
   public RpcRetryingCallerFactory(Configuration conf) {
```

Comment:
this class is also IA.Private, so we can try not to add to the overloads here at least. I thought it wasn't IA.Private, but now that I see it is we can make a breaking change here if need be.

Reply:
this one is just used in 18 spots (mostly tests) I can update